### DISCUSSION TITLE
Contributing to Rocket

### DISCUSSION THREAD

**Discussion by DvrDania** [rwf2/Rocket#1837] [Questions]
Hi,
I'm new in rust and I really like Rocket. I wanna contribute but as a beginner in rust I find it hard to find things because of its huge codebase.

I'm really determined about it.

What should I do? Where should I start from?

**Comment by MikailBag** [comment]
Just in case you haven't heard about it, I would recommend using a [Rust Analyzer](rust-analyzer.github.io). It supports many popular text editors and IDEs (including VSCode) and greatly simplifies developing Rust projects. For example, it supports `Go to definition`, `Code completion` and `Find references`. These features really simplify navigation in unknown code bases.

**Reply by DvrDania** [reply]
Yes I used it and it's very helpful.
Now maybe I should try to fix an issue? I never contributed in any open source project before.
Do I just pick an issue and work on it?
